composer.json

```diff
@@ -21,9 +21,10 @@
 		"nette/di": "~2.3@dev",
 		"nette/http": "~2.3@dev",
 
-		"kdyby/curl-ca-bundle": "~1.0",
+		"composer/ca-bundle": "~1.0",
 		"ext-curl": "*",
-		"ext-json": "*"
+		"ext-json": "*",
+		"symfony/process": "~3.1"
```

Comment:
docs say `but can be ignored on PHP 5.5.6+`
`nette/application` etc. already requires php 5.6. It should be removed I think, not neeeded